src/server.c

```diff
@@ -159,6 +159,10 @@ struct redisServer server; /* Server global state */
  *              delay its execution as long as the kernel scheduler is giving
  *              us time. Note that commands that may trigger a DEL as a side
  *              effect (like SET) are not fast commands.
+ * 
+ * can-replicate: Command may produce replication traffic but has it's own
+ *                explicit checks to make sure it's not called on a replcia.
+ *                This is only used for client pause to allow non-write commands.
  *
```

Comment:
I'm pretty sure the meaning of can/may has degraded in the US language, but I like the sound of may more anyways. Might is also the future tense, so that doesn't feel right. 

I'm going to intentionally not conflate it with write commands, since this is any commands that replicate data over the replication stream. We've talked about pulling out a separate channel for non-writes, and at that time we might revisit this.